### ISSUE TITLE
Investigate Flink-on-k8s operator

### ISSUE THREAD

**Issue by idomingu** [giros-dit/semantic-data-aggregator#23]
## Description

Spotify has developed a K8s operator, called [flink-on-k8s](https://github.com/spotify/flink-on-k8s-operator) that enables managing LCM of Flink applications. The operator introduces the concept of Flink Cluster as CRD in Kubernetes. It allows setting up "session-based" and "job-based" Flink clusters in Kubernetes – the second type refers to the Application mode. 

Additionally, this operators allows for submitting Application-mode jobs that may build on JAR files or Python scripts that leverage PyFlink API.

In summary, this operator can a solution to our problems of orchestrating Flink jobs executed in the Application mode – the standard mechanism is by using the Flink CLI.
